test/typedarray.js

```diff
@@ -6,15 +6,15 @@ module.exports = require('./common').runTest(test);
 
 function test (binding) {
   const testData = [
-    ['int8', Int8Array],
-    ['uint8', Uint8Array],
-    ['uint8_clamped', Uint8ClampedArray],
-    ['int16', Int16Array],
-    ['uint16', Uint16Array],
-    ['int32', Int32Array],
-    ['uint32', Uint32Array],
-    ['float32', Float32Array],
-    ['float64', Float64Array]
+    ['int8', Int8Array, 1],
+    ['uint8', Uint8Array, 1],
+    ['uint8_clamped', Uint8ClampedArray, 1],
+    ['int16', Int16Array, 2],
+    ['uint16', Uint16Array, 2],
+    ['int32', Int32Array, 4],
+    ['uint32', Uint32Array, 4],
+    ['float32', Float32Array, 4],
+    ['float64', Float64Array, 8]
```

Comment:
Sizes accordingly to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray#typedarray_objects